librecval/extract_phrases.py

```diff
@@ -50,34 +52,47 @@ class MissingMetadataError(RuntimeError):
     """
 
 
-class InvalidTextGridName(RuntimeError):
+class InvalidFileName(RuntimeError):
     """
-    Raised when the TextGrid name does not follow an appropriate pattern.
+    Raised when the ELAN name does not follow an appropriate pattern.
     """
 
 
+class MissingTranslationError(RuntimeError):
+    """
+    Raised when the 'English (word)' and 'English (sentence)' tiers
+    can not be found in a .eaf file suggesting that the word or phrase
+    does not have an existing translation
+    """
+
+    # TODO: this is a weird assumption to make. So far it hasn't been an issue though?
+
+
 # ########################################################################## #
 
 
-class RecordingInfo(NamedTuple):
+class Segment(NamedTuple):
     """
-    All the information you could possible want to know about a recorded
-    snippet.
+    Stores an audio segment extracted from a .eaf file
     """
 
-    session: SessionID
+    translation: str  # in English
+    transcription: str  # in Cree
```

Comment:
Two similarly-named variables with comments to help distinguish them? Sometimes you can move the comment into the variable name.

```suggestion
    english_translation: str
    cree_transcription: str
```